windows-driver-docs-pr/image/--hello-world---installation-file.md

```diff
@@ -60,7 +60,7 @@ UI Class ID={4DB1AD10-3391-11D2-9A33-00C04FA36145}
 HKR,,HardwareConfig,1,1
 HKR,,USDClass,,"{7C1E2309-A535-45b1-94B3-9A020EE600C6}"
 HKCR,CLSID\{7C1E2309-A535-45b1-94B3-9A020EE600C6},,,"Hello World WIA Minidriver"
-HKCR,CLSID\{7C1E2309-A535-45b1-94B3-9A020EE600C6}\InProcServer32,,,%11%\hellowld.dll
+HKCR,CLSID\{7C1E2309-A535-45b1-94B3-9A020EE600C6}\InProcServer32,,,%%SystemRoot%%\hellowld.dll
```

Comment:
Your change of the DefaultDestDir to 13 (run from driverstore) means this file won't exist at %systemroot%\hellowld.dll.  This needs to be %13%\hellowld.dll